docs/topics/OAuth2.md

```diff
@@ -56,68 +43,232 @@ Scopes provide access to certain resources of a user's account. Your API client
 | webhook.incoming | this generates a webhook that is returned in the oauth token response for authorization code grants |
 
 >info
-> Unlike the rest of the scopes, `guilds.join` requires you to have a bot account linked to your application and can only be used to join users to guilds which your bot services.
+>Unlike the rest of the scopes, `guilds.join` requires you to have a bot account linked to your application and can only be used to join users to guilds which your bot services.
```

Comment:
technically `bot` scope also requires a bot account, but that seems a little on-the-nose, so i'm not sure if it needs to be mentioned